Assets/Scripts/Waypoint Scripts/WaypointProperties.cs

```diff
@@ -144,14 +163,22 @@ public void SetLineCollider()
         {
             Vector3 endpoint = prevPoint.transform.position;
 
+            ComputeWorldScaleActual();
+
             lineCollider.transform.parent = LineProperties.transform;
-            lineCollider.radius = controller.GetComponent<MapInteractions>().actualScale.y / 50;
+            
+            // Code in WaypointSystemUpgrade
+            lineCollider.radius = WorldScaleActual.y / 50;
```

Comment:
will lineCollider.radius change with zoom in / out post placement?

Reply:
yes